lib/LaTeXML/Package/TeX.pool.ltxml

```diff
@@ -4453,8 +4454,9 @@ sub revertScript {
     $l++; shift(@t); }
   while (@t && $l) {
     my $t = shift(@t);
-    if    ($t->defined_as(T_BEGIN)) { $l++; }
-    elsif ($t->defined_as(T_END))   { $l--; } }
+    if (ref $t eq 'LaTeXML::Core::Token') {    # jump over pre-wrapped Tokens, see arXiv:2210.11051
```

Comment:
Interesting that the leading comment ends with "Worry about list of lists". That `->revert` would yield a list of both `Token` and `Tokens` almost seems like the real bug. Either way, your patch doesn't protect against *leading* `Tokens`. It's also unclear that simply skipping over `Tokens` is the right thing; what if there are unbalanced or leading or trailing braces in there?

I'd think a simpler, though slightly inefficient, approach would be
```
my @tokens = Tokens($script->revert)->unlist;
```

Reply:
Excellent suggestion - and indeed more correct. 

Done: Tested with two articles, seems to work well.